projects/vdk-plugins/vdk-control-api-auth/tests/test_core_auth.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+"""
```

Comment:
why file named core? is there something in addition to test?

Reply:
Yes, there will be additional tests for the different authentication flows.